lib/util/miq_apache/miq_apache.rb

```diff
@@ -125,7 +125,7 @@ def self.status(full = true)
     end
 
     def self.version
-       MiqUtil.runcmd("rpm -qa --queryformat '%{VERSION}' httpd")
+      MiqUtil.runcmd("rpm -qa --queryformat '%{VERSION}' #{MiqApache.package_name}")
```

Comment:
You know,  you got me thinking.   Should Apache just be a top level thing in LinuxAdmin? Maybe a subclass of service? Doesn't have to be in this PR,  but I feel like external things should stay external.